apps/allocations/app/components/Card/Budget.js

```diff
@@ -1,6 +1,7 @@
 import PropTypes from 'prop-types'
 import React from 'react'
 import styled from 'styled-components'
+import BN from 'bignumber.js'
```

Comment:
This seems dishonest. If we call it `BN`, let's `import { BN } from 'web3-utils'`. Otherwise, let's call it `BigNumber`